app/models/task.py

```diff
@@ -1,6 +1,37 @@
-from flask import current_app
 from app import db
+from flask import current_app
+from sqlalchemy import DateTime
```

Comment:
since we are already importing db, we don't need to import DateTime separately. It already comes in db, like `db.String`, `db.Column`, etc.

```suggestion

```